pkg/storage/bloom/v1/bloom_tester.go

```diff
@@ -0,0 +1,156 @@
+package v1
+
+import (
+	"github.com/prometheus/prometheus/model/labels"
+
+	"github.com/grafana/loki/pkg/logql/syntax"
+	"github.com/grafana/loki/pkg/storage/bloom/v1/filter"
+)
+
+type BloomTest interface {
+	Test(bloom filter.Checker) bool
```

Comment:
nit: `Test()` does not give any clue about what the returned `true` or `false` means.

`BloomTest` is a [predicate](https://en.wikipedia.org/wiki/Predicate_%28mathematical_logic%29) of `Request`.

```go
type MatchesBloom interface {
    ...
}
```

or

```go
struct BloomTest interface {
    Matches(bloom filter.Checker) bool
    ...
}
```